src/Discord.Net.Commands/SlashCommands/CommandRegistration/ExistingCommandOptions.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Discord.SlashCommands
+{
+    public enum ExistingCommandOptions
+    {
+        OVERWRITE,
+        KEEP_EXISTING
+    }
```

Comment:
[By convention](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum), enums should be written in pascal case.

```suggestion
    public enum ExistingCommandOptions
    {
        Overwrite,
        KeepExisting
    }
```